.circleci/config.yml

```diff
@@ -1,4 +1,4 @@
-version: 2
+version: 2.1
```

Comment:
This is no longer needed and is an artifact of when I was going to use an `orb` for the codecov. We can revert to v2 if preferred.